dbt/models/reporting/docs.md

```diff
@@ -32,14 +32,54 @@ Feeds public reporting assets.
 # res_report_summary
 
 {% docs table_res_report_summary %}
-Materialized version of `reporting.vw_res_report_summary`.
+Aggregates statistics on characteristics, classes, AVs, and sales
+by assessment stage, property groups, year, and various geographies.
+Feeds public reporting assets.
 
-Materialized to speed up queries for Tableau.
+Materialized once per day to speed up queries for Tableau.
+
+## Nuance
+
+- Model and assessment values are gathered independently and
+  aggregated via a union rather than a join, so it's important to keep in mind
+  that years for model and assessment stages do NOT need to match, i.e. we can
+  have 2023 model values in the table before there are any 2023 assessment
+  values to report on. Sales are added via a lagged join, so `sale_year` should
+  always be `year - 1`. It is also worth nothing that "model year" is
+  incremented by 1 solely for the sake of reporting in this table, meaning that
+  models with a `meta_year` value of 2022 in `model.assessment_pin` will
+  populate the table with a value of 2023 for `year`.
 
 **Primary Key**: `year`, `geography_type`, `geography_id`, `assessment_stage`,
 `property_group`
 {% enddocs %}
 
+# res_report_summary_sales_input
+
+{% docs table_res_report_summary_sales_input %}
+Input table for `reporting.res_report_summary` that produces the raw
+sales data that `res_report_summary` aggregates.
+
+We split these input data out into a separate table to speed up query time for
+`res_report_summary`, since otherwise it needs to rerun the query logic
+for every possible geography and reporting group combination.
+
+**Primary Key**: `pin`, `doc_no`
+{% enddocs %}
+
+# res_report_summary_values_input
+
+{% docs table_res_report_summary_values_input %}
+Input table for `reporting.res_report_summary` that produces the raw
+characteristic and value data that `res_report_summary` aggregates.
+
+We split these input data out into a separate table to speed up query time for
+`res_report_summary`, since otherwise it needs to rerun the query logic
+for every possible geography and reporting group combination.
+
+**Primary Key**: `pin`, `year`
```

Comment:
Note that, due to the problem I shared in our chat, this view isn't actually _truly_ unique by `pin` and `year` since there's one PIN in the 2023 condo model that has two rows in `model.assessment_pin`. I think that's a minor detail and I doubt anyone will need to rely on the uniqueness of this table, however, so I left that context out of the docs -- happy to put it back in if you think it's appropriate.

Reply:
That's fine. I'll debug the dupe PIN. I'm going to add a follow-up issue for data sets on the `model` DB assets.